bears/haml/HamlLintBear.py

```diff
@@ -0,0 +1,30 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GemRequirement import GemRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable='haml-lint',
+        output_format='regex',
+        output_regex=r'(?P<filename>\S+):(?P<line>\d+) \[(?P<severity>W|E)\] '
+                     r'(?P<message>.*)',
+        severity_map={'E': RESULT_SEVERITY.MAJOR,
+                      'W': RESULT_SEVERITY.NORMAL},
+        use_stdin=True)
+class HamlLintBear:
+    """
+    Checks code using ``haml-lint``. To
+    ensure clean and readable HAML code
```

Comment:
Refactor the sentence to `Checks code using ``haml-lint`` to ensure clean and readable HAML code.`